docs/tremor-query/operators.md

```diff
@@ -137,12 +137,19 @@ This operator is deprecated please use `qos::backpressure` instead.
 
 The backpressure operator is used to introduce delays based on downstream systems load. Longer backpressure steps are introduced every time the latency of a downstream system reached `timeout`, or an error occurs. On a successful transmission within the timeout limit, the delay is reset.
 
+The operator supports two modes:
+
+- `discard` - the standard, when backpressure is triggered it will discard new messages. This is designed to fulfill the need of low transport latency at the cost of loss.
+- `pause` - it will trigger a circuit breaker and ask the sources that send it data to stop sending additional events. This is designed to deal with situations where loss isn't an option - the guarnatee of lossiness is then dependant on the source and how it can handle circuit breaker events.
```

Comment:
```suggestion
- `pause` - it will trigger a circuit breaker and ask the sources that send it data to stop sending additional events. No Event is discarded by the backpressure operator. This is designed to deal with situations where losing events is not an option - but the garuantee of losslessness depends on the source and how it can handle circuit breaker events.
```

Reply:
If we mention here that it depends on the source and how it handles circuit breaker events, we should have it documented somewhere. I dont think we do, tbh.